### PR TITLE
Add support for integer where appropriate.

### LINKED ISSUE #16: Add support for integers
In the [Thing Description specification](https://iot.mozilla.org/wot/), the property type can be one of: null, boolean, object, array, number, integer or string. However, in the [schema specification](https://iot.mozilla.org/schemas), integer is never used, only number. Most of the properties in schemas that have `type: number` could be `type: number/integer`. 

_Originally from mozilla-iot/gateway#1467_

### PR DESCRIPTION
**Pull request by mrstegeman**
Fixes #16

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -685,11 +687,11 @@ <h3>data</h3>
         <table>
           <tr>
             <th>type</th>
-            <td>number</td>
+            <td>number/integer</td>
           </tr>
           <tr>
             <th>unit</th>
-            <td>degree celcius</td>
```
Comment (benfrancis) [inline index.html]: [According to Wikipedia](https://en.wikipedia.org/wiki/International_System_of_Units#Derived_units) and [bipm.org](https://www.bipm.org/en/publications/si-brochure/kelvin.html) the long form of the SI unit is "degree celcius", not "celcius".

File: index.html [hunk 2]
```diff
@@ -546,7 +547,7 @@ <h2>BrightnessProperty</h2>
         <table>
           <tr>
             <th>type</th>
-            <td>number</td>
+            <td>integer</td>
```
Comment (benfrancis) [inline index.html]: Might there not be a use case where someone wants to set brightness to 10.5%? Should this maybe be number/integer?

File: index.html (hunk 1 above)
Comment (mrstegeman) [reply index.html]: I was trying to maintain internal consistency across both of our specs. If we want `degree celsius`, that's fine. We just need to change it in several other places.

**Comment by mrstegeman** [comment]
@benfrancis Fixed both issues.

**Pull request merged**